src/sage/rings/polynomial/multi_polynomial_ring_base.pyx

```diff
@@ -26,6 +26,7 @@ from sage.rings.polynomial import polynomial_ring
 from sage.rings.polynomial.term_order import TermOrder
 from sage.rings.polynomial.polynomial_ring_constructor import (PolynomialRing,
                                           polynomial_default_category)
+from sage.rings.polynomial.polydict import ETuple
```

Comment:
maybe use cimport here if it works

Reply:
should I also fix the linter complaints, or is this better in a separate pull request?

Reply:
Oh sorry, the linter is misreporting, because this is cython not python